### PR TITLE
Add description to static pages in v5 docs

### PR DESCRIPTION
**Pull request by fflaten**
Adds a description front matter to all v5 statics pages. These are used as description in search results etc.

Related #2

### CODE REVIEW

**Comment by fflaten** [comment]
Please review 🙂

File: docs/assertions/assertions.mdx [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ---
 id: assertions
 title: Assertion Reference
+description: Introduction to the assertion operators that comes built-in with Pester to get you started with the most common assertions
```
Comment (bravo-kernel) [inline docs/assertions/assertions.mdx:4]: > that come

File: docs/assertions/should-command.mdx [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ---
 title: Should
+description: Introduction to the Should command that's used to define assertions that fails or passes the test
```
Comment (bravo-kernel) [inline docs/assertions/should-command.mdx:3]: > that fail or pass the test

File: docs/contributing/reporting-issues.mdx [hunk 3]
```diff
@@ -2,6 +2,7 @@
 id: reporting-issues
 title: Reporting Issues
 sidebar_label: Reporting Issues
+description: Found an issue with Pester? Check out this guide for how to create an issue and provide us feedback so we can get help you
```
Comment (bravo-kernel) [inline docs/contributing/reporting-issues.mdx:5]: > so we can help you

File: docs/migrations/breaking-changes-in-v5.mdx [hunk 4]
```diff
@@ -1,6 +1,7 @@
 ---
 id: breaking-changes-in-v5
 title: Breaking changes in v5
+description: Pester v5 was a major upgrade with a new runtime. This lead to some breaking changes in syntax and features. See this list of breaking changes when upgrading to Pester v5
```
Comment (bravo-kernel) [inline docs/migrations/breaking-changes-in-v5.mdx:4]: >this leads

File: docs/usage/configuration.mdx [hunk 5]
```diff
@@ -1,5 +1,6 @@
 ---
 title: Configuration
+description: How to configure Pester's behavior, options and features
```
Comment (bravo-kernel) [inline docs/usage/configuration.mdx:3]: Unsure but.. `Pesters behavior`?

File: docs/usage/file-placement-and-naming.mdx [hunk 6]
```diff
@@ -1,5 +1,6 @@
 ---
 title: File placement and naming
+description: Tips on how to name and structure your Pester test files and categorize your tets
```
Comment (bravo-kernel) [inline docs/usage/file-placement-and-naming.mdx:3]: >tests

File: docs/usage/vscode.mdx [hunk 7]
```diff
@@ -1,5 +1,6 @@
 ---
 title: VSCode
+description: See how Visual Studio Code and it's extension helps you run and debug your Pester tests directly inside the editor
```
Comment (bravo-kernel) [inline docs/usage/vscode.mdx:3]: > its?

**Review by bravo-kernel — CHANGES_REQUESTED** [review]
Looking very good. I think I found some grammar but merging after that 💯

**Comment by fflaten** [comment]
Thanks. Yeah it was late 😅 Will cleanup later today.

File: docs/usage/configuration.mdx (hunk 5 above)
Comment (fflaten) [reply docs/usage/configuration.mdx:3]: In english I believe it's "Owner's item", but I'll rephrase it :)

File: docs/migrations/breaking-changes-in-v5.mdx (hunk 4 above)
Comment (fflaten) [reply docs/migrations/breaking-changes-in-v5.mdx:4]: rewrote it 👍

File: docs/usage/configuration.mdx (hunk 5 above)
Comment (bravo-kernel) [reply docs/usage/configuration.mdx:3]: You could very well be correct, after re-reading the grammar rules I am now very confused :-D

**Comment by bravo-kernel** [comment]
Please let me know when this one is good to merge.

File: docs/assertions/assertions.mdx (hunk 1 above)
Comment (fflaten) [reply docs/assertions/assertions.mdx:4]: Rewrote it 👍

**Review by bravo-kernel — APPROVED** [review]

**Comment by bravo-kernel** [comment]
Was about to merge but it seems there is a conflict. Could you please look into that?

**Comment by fflaten** [comment]
Rebased. Just a conflict with the whitespace cleanup

**Comment by bravo-kernel** [comment]
Awesome job, thanks again !

You might have noticed by now that I am not merging any of the Powershell-content Pull Requests. We will have to wait for a review by @nohwnd for that, I am no pro.

**Pull request merged**